src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md

```diff
@@ -2,36 +2,36 @@
 
-Fun Fact. The Netherlands started suffering from the lack of wood even earlier than England. However, instead of inventing coke, the Dutch just switched to using peat. You might imagine the taste of Dutch beer in the 16th century.
+Fun Fact - the Netherlands started suffering from a lack of wood even earlier than England. However, wihtout the innovation of coke, the Dutch switched to using peat. You might imagine the taste of Dutch beer in the 16th century.
 
-Those two factors, namely hard water and pale malt, led to the birth of the ‘Burton Ale’, which was a thick, strong, intensively hopped sweet beer. It was not literally pale: malt was additionally roasted or caramelized. Today, we would call this beer ‘amber’.
+Those two factors, namely hard water and pale malt, led to the birth of the ‘Burton Ale’, which was a thick, strong, intensively hopped sweet beer. However, it was not literally pale. Malt was additionally roasted or caramelized. Today, we would call this beer ‘amber’.
 
-The history of Burton Ale consists of rises and falls, half a century in-between each. The period of prosperity based on the Baltic trade did not last long. In 1783, Russian authorities imposed a 300 percent tax on beer imports; then went the continental blockade of Napoleonian time and the annexation of Poland by Russia, which had closed the Polish market for English brewers. Finally, in 1822, a new Russian customs tariff was introduced that effectively banned all sorts of imports (including beer) from Britain. As a result, at the beginning of the 19th century, the Burton brewing industry deteriorated (four out of 15 breweries were closed, several others sold[ref:brewing-in-burton:29]()) — just to make the rebirth from the ashes (story of which we will tell in the next chapter) even more spectacular.
+The history of Burton Ale consists of rises and falls, half a century passing between each. The period of prosperity based on the Baltic trade did not last long. In 1783, Russian authorities imposed a 300 percent tax on beer imports. They were then obliged to follow the *Continental System* during the Napoleonic era. The annexation of Poland by Russia  had also closed the Polish market for English brewers. Finally, in 1822, a new Russian customs tariff was introduced that effectively banned various and sundry imports (including beer) from Britain. As a result, at the beginning of the 19th century, the Burton brewing industry deteriorated (four out of 15 breweries were closed, several others sold[ref:brewing-in-burton:29]()) — just to make the rebirth from the ashes (the story of which shall be told in the next chapter) even more spectacular.
```

Comment:
I would rephrase it. It was Napoleon who tried to enforce the blockade, and once some territories (including Russian ones) were falling into his control, they migh have joined the blockade. Too complex sentence to explain it, so I would rather leave it simply 'the Napolean-inspired Continental Blockade against British goods' or smth. Also, I would prefer 'blockade' over 'system' on the same grounds, as 'system' will require explanations.

Reply:
Russia was subdued by Napoleon by 1803, Really the whole continent. The continental system is the historic term, though without a Navy it cut France off from things it needed, like sugarcane. i made the change.